proxy/http2/Http2ConnectionState.cc

```diff
@@ -1054,7 +1055,7 @@ Http2ConnectionState::send_a_data_frame(Http2Stream *stream, size_t &payload_len
     return HTTP2_SEND_A_DATA_FRAME_NO_PAYLOAD;
   }
 
-  if (stream->is_body_done() && payload_length < available_size) {
+  if (stream->is_body_done() && payload_length <= available_size) {
```

Comment:
Yeah, I updated the patch with <= so that it would set the END_STREAM on a 0 length frame.  Thanks for the feedback.